app/models/krikri/original_record.rb

```diff
@@ -42,7 +42,8 @@ def load(identifier)
           record = new(identifier)
         end
 
-        raise "No #{self} found with id: #{identifier}" unless record.exists?
+        raise NameError, "No #{self} found with id: #{identifier}" unless 
```

Comment:
That feels awkward to me, too. I'm not sure `NotFound` is what we're looking for, here.

Probably we want our own error type... but that raises the question: where does the custom error sit in the hierarchy? Underneath `NameError`.

This line of thinking is why I pushed it up and asked for reviewer input. :smile: